packages/integrations/vercel/README.md

```diff
@@ -281,6 +281,28 @@ export default defineConfig({
 });
 ```
 
+### streaming
+
+**Type:** `boolean`<br>
+**Available for:** Serverless
+
+Determines whether or not the serverless function will stream its response to the browser, sending components as they are rendered.
+
+The default value is `false`. Set this value to `true` in your Vercel adapter configuration to enable streaming.
```

Comment:
```suggestion
Use this property to enable Astro's HTML streaming.
```
I kind of want something like this now, because the docs really do imply this happens by default. (And yes, this seems to be more consistent with the others.)

What do you think about something like this?

Reply:
Hmm it won't be limited to HTML. Although the non-HTML cases would be where the user is creating their own streaming responses from scratch.

Reply:
OK, so I'll tell you where my little stumbling blocks are and YOU propose what works!  :smile: 

Grammatically:

- "**the** serverless function" sounds jarring out of place. I never like a "the" if we haven't defined something yet. If you can be more descriptive and avoid the implied context of a "the", that would go a long way.

Philosophically:

- I really don't like that regular docs say that we do HTML streaming and apparently, sometimes we don't! So, I'd be happier from a narrative standpoint if this default were set to true (can we do that? can we build it that way?) and then this configuration turns it OFF. Then the story matches the other adapters, and we can say that you CAN turn it off to do x, y, and z if we want to give use cases that are not... just how our stuff is supposed to work? I feel we can get away with being less define-y here in that case, and say you can remove and either not use, roll your own etc.

Reply:
So, _the_ serverless function is the particular one created when you deploy to Vercel, at least intended to be that one.

> So, I'd be happier from a narrative standpoint if this default were set to true

I agree, it's a documented feature. We could probably make a major release to change the default **very soon** (but not right away), so that streaming is always the case. And I think it is a good idea for another major release after to remove the option altogether.

> Then the story matches the other adapters

Netlify never streams either 😶

Reply:
> can we build it that way?
Yes :D

Reply:
It is now built that way. Streaming with no configuration to opt-out, just the way it should be.

Reply:
> Netlify never streams either 😶

What even is happening?? The cake is a lie!  :sweat_smile: 

So, I don't want anyone's project to break with this change, of course! Aligning this with docs is helpful for me, since that's the story we tell pretty clearly, but if we've been misinformed then we can change on our end, too.

Reply:
Breaking is not a concern, just that the change is going to be more _visible_ than what someone might expect from a minor or patch release. So a major release is important more because the user needs to be given notice about what's happening.